### PR TITLE
Update sites.map for INC12692585 and INC12659293

### PR DESCRIPTION
**Pull request by lizmcguireBU**
Route www.bu.edu/systems/wp-assets/ for Telegraph Nelnet form files outside WP (INC12692585)
Route www.bu.edu/urbanclimate/wp-assets/ for Telegraph Majordomo form files outside WP (INC12659293)

### CODE REVIEW

File: landscape/prod/maps/sites.map [hunk 1]
```diff
@@ -2005,6 +2006,7 @@ _/univcomp content ;
 _/univcomputer content ;
 _/universe content ;
 _/ur content ;
+_/urbanclimate/wp-assets/  content ;
```
Comment (Adam42) [inline landscape/prod/maps/sites.map]: Trailing slash is not needed. Not sure if it's detrimental but to maintain consistency should be removed.

**Review by Adam42 — CHANGES_REQUESTED** [review]
Minor change to remove trailing slash

File: landscape/prod/maps/sites.map (hunk 1 above)
Comment (lizmcguireBU) [inline landscape/prod/maps/sites.map]:
```suggestion
_/urbanclimate/wp-assets  content ;
```

**Review by Adam42 — APPROVED** [review]
LGTM

**Pull request merged**